xmpp/browser.py

```diff
@@ -61,65 +62,80 @@ class Browser(PlugIn):
+        features = [NS_DISCO_INFO, NS_DISCO_ITEMS, NS_MUC,NS_REGISTER, NS_SEARCH, NS_TIME, NS_VERSION]
+        info = {'ids':ids, 'features':features}
+        # info['xdata'] = xmpp.protocol.DataForm() # XEP-0128
+        b.setDiscoHandler({'items':[], 'info':info})
     """
+
     def __init__(self):
-        """Initialises internal variables. Used internally."""
+        """Initializes internal variables. Used internally."""
         PlugIn.__init__(self)
-        DBG_LINE='browser'
-        self._exported_methods=[]
-        self._handlers={'':{}}
+        DBG_LINE = 'browser'
+        self._exported_methods = []
+        self._handlers = {'': {}}
 
     def plugin(self, owner):
         """ Registers it's own iq handlers in your application dispatcher instance.
             Used internally."""
-        owner.RegisterHandler('iq',self._DiscoveryHandler,typ='get',ns=NS_DISCO_INFO)
-        owner.RegisterHandler('iq',self._DiscoveryHandler,typ='get',ns=NS_DISCO_ITEMS)
+        owner.RegisterHandler(
+            'iq', self._DiscoveryHandler, typ='get', ns=NS_DISCO_INFO)
+        owner.RegisterHandler(
+            'iq', self._DiscoveryHandler, typ='get', ns=NS_DISCO_ITEMS)
 
     def plugout(self):
         """ Unregisters browser's iq handlers from your application dispatcher instance.
             Used internally."""
-        self._owner.UnregisterHandler('iq',self._DiscoveryHandler,typ='get',ns=NS_DISCO_INFO)
-        self._owner.UnregisterHandler('iq',self._DiscoveryHandler,typ='get',ns=NS_DISCO_ITEMS)
+        self._owner.UnregisterHandler(
+            'iq', self._DiscoveryHandler, typ='get', ns=NS_DISCO_INFO)
+        self._owner.UnregisterHandler(
+            'iq', self._DiscoveryHandler, typ='get', ns=NS_DISCO_ITEMS)
 
-    def _traversePath(self,node,jid,set=0):
+    def _traversePath(self, node, jid, set=0):
         """ Returns dictionary and key or None,None
             None - root node (w/o "node" attribute)
             /a/b/c - node
             /a/b/  - branch
             Set returns '' or None as the key
             get returns '' or None as the key or None as the dict.
             Used internally."""
-        if jid in self._handlers: cur=self._handlers[jid]
+        if jid in self._handlers:
+            cur = self._handlers[jid]
         elif set:
-            self._handlers[jid]={}
-            cur=self._handlers[jid]
-        else: cur=self._handlers['']
-        if node is None: node=[None]
-        else: node=node.replace('/',' /').split('/')
+            self._handlers[jid] = {}
+            cur = self._handlers[jid]
+        else:
+            cur = self._handlers['']
+        if node is None:
+            node = [None]
+        else:
+            node = node.replace('/', ' /').split('/')
         for i in node:
-            if i!='' and i in cur: cur=cur[i]
-            elif set and i!='': cur[i]={dict:cur,str:i}; cur=cur[i]
-            elif set or '' in cur: return cur,''
-            else: return None,None
```

Comment:
Just a random drive-by comment. 
Original xmpppy code was totally off the common styling rules, but it still had a certain style :)

Basically it was: the whole file must fit on my (vertically oriented) 19" screen. Therefore I tried to stay on the same line as much as possible.

Reply:
Actually, I'd recommend to not take this change if there are still any other forks out there that have valuable, but not-yet-merged changes. Whitespace changes might be very disrupting to carrying diffs around.